migrator/migrator.go

```diff
@@ -179,22 +179,25 @@ func (p *PouchMigrator) PreMigrate(ctx context.Context, takeOverContainer bool)
 			return err
 		}
 
+		var imageName string
 		// meta.Image maybe a digest, we need image name.
 		image, err := p.dockerd.ImageInspect(meta.Image)
-		if err != nil {
+		if err != nil && strings.Contains(err.Error(), "No such image") {
+			// image used by docker container already is deleted
+			imageName = meta.Image
+			logrus.Errorf("image %s used by %s already is already deleted", meta.Image, c.ID)
+		} else if err != nil {
 			return err
-		}
-
-		fmt.Printf("image: %v\n", image)
-
-		var imageName string
-		if len(image.RepoTags) > 0 {
-			imageName = image.RepoTags[0]
-		} else if len(image.RepoDigests) > 0 {
-			imageName = image.RepoDigests[0]
 		} else {
-			return fmt.Errorf("failed to get image %s: repoTags is empty", meta.Image)
+			if len(image.RepoTags) > 0 {
```

Comment:
please combine this `if` to the upper level to be `else if`. This could reduce indent.